keel-core/src/main/kotlin/com/netflix/spinnaker/keel/serialization/ObjectMappers.kt

```diff
@@ -19,17 +19,30 @@ import de.huxhorn.sulky.ulid.ULID
 import java.text.SimpleDateFormat
 import java.util.TimeZone
 
+class ObjectMapperSingleton private constructor() {
+  companion object {
+    val instance: ObjectMapper by lazy { ObjectMapper() }
+  }
+}
+
+class YamlMapperSingleton private constructor() {
+  companion object {
+    val instance: YAMLMapper by lazy { YAMLMapper() }
+  }
+}
+
 /**
  * Factory method for [ObjectMapper]s configured how we like 'em.
  */
 fun configuredObjectMapper(): ObjectMapper =
-  ObjectMapper().configureMe()
+  ObjectMapperSingleton.instance.configureMe()
```

Comment:
This will still call `configureMe` each time. Feels like you could roll that into the creation of the instance

Reply:
Good point. Will fix.